google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/GapicSpannerRpc.java

```diff
@@ -282,6 +282,10 @@ public static GapicSpannerRpc create(SpannerOptions options) {
   }
 
   public GapicSpannerRpc(final SpannerOptions options) {
+    this(options, true);
```

Comment:
The changes in this file look very big (on GitHub), but are in reality quite small:
1. An additional package-private constructor for test purposes has been added that allows creating a `GapicSpannerRpc` without creating and initializing the gRPC stubs. This allows testing methods that do not communicate with any backend without the need to setup a mock server, and without the additional execution time of initializing and closing the stubs.
2. The initialize and close methods contain an if-statement to check whether to actually initialize / close stubs.